### PR TITLE
fix #60332 voenhronika.ru

### LINKED ISSUE #60332: voenhronika.ru
### Issue URL (Ads)
 [https://voenhronika.ru/publ/vtoraja_mirovaja_vojna_germanija_khronika/chto_esli_by_gitler_ne_rodilsja_scenarij_takogo_alternativnogo_khoda_istorii_vygljadit_bolee_uzhasajushhim_2020/23-1-0-7501](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fvoenhronika.ru%2Fpubl%2Fvtoraja_mirovaja_vojna_germanija_khronika%2Fchto_esli_by_gitler_ne_rodilsja_scenarij_takogo_alternativnogo_khoda_istorii_vygljadit_bolee_uzhasajushhim_2020%2F23-1-0-7501)
### Comment
Username: @spok1
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by stanislav-atr**
#60332

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -47,6 +47,9 @@ life.pravda.com.ua##.net-wrap
 tweaktown.com##div[class^="sharedesktop"]
 ichannela.com##.sns > a[href^="javascript:openSendNews"]:not([href="javascript:openSendNews('url');"])
 biomedya.com##.share-url
+voenhronika.ru##aside.sidebar > section.sideboxrek
+voenhronika.ru##a[href^="https://twitter.com/voenhronika"]
+voenhronika.ru##aside.sidebar > section:last-child
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt:52]: Also empty without AdGuard. Also better avoid using selector like this.

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -47,6 +47,9 @@ life.pravda.com.ua##.net-wrap
 tweaktown.com##div[class^="sharedesktop"]
 ichannela.com##.sns > a[href^="javascript:openSendNews"]:not([href="javascript:openSendNews('url');"])
 biomedya.com##.share-url
+voenhronika.ru##aside.sidebar > section.sideboxrek
+voenhronika.ru##a[href^="https://twitter.com/voenhronika"]
```
[the remark this answers is not in the document]
Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt:51]: And please restore the original appearance of the twitter button.

File: SocialFilter/sections/specific.txt [hunk 3]
```diff
@@ -47,6 +47,9 @@ life.pravda.com.ua##.net-wrap
 tweaktown.com##div[class^="sharedesktop"]
 ichannela.com##.sns > a[href^="javascript:openSendNews"]:not([href="javascript:openSendNews('url');"])
 biomedya.com##.share-url
+voenhronika.ru##aside.sidebar > section.sideboxrek
```
[the remark this answers is not in the document]
Comment (zloyden) [reply SocialFilter/sections/specific.txt:50]: According to their code, these blocks might be used for ads, but without AdGuard I just see leftovers. I guess we can add it to Annoyances for now.

File: SocialFilter/sections/specific.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (zloyden) [reply SocialFilter/sections/specific.txt:51]: @Alex-302 I don't think that we should restore twitter element. It has a recommendation `Follow`, also we have similar rules in our filters `##.twitter-follow-button`.

[the remark this answers is not in the document]
Comment (Alex-302) [reply SocialFilter/sections/specific.txt]: agree. @stanislav-atr

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**